packages/main/src/ListItemBase.ts

```diff
@@ -42,7 +42,8 @@ class ListItemBase extends UI5Element implements ITabbable {
 	/**
 	 * Defines whether the item is movable.
 	 * @default false
-	 * @private
+	 * @public
+	 * @since 2.0.0
```

Comment:
Actually, this will be shown for every item that derives from the ListItemBase, like notifications which should have it. We probably need to make it public just in the StandardListItem, CustomListItem and TreeItemBase